mediachain/indexer/mc_ingest.py

```diff
@@ -295,7 +296,7 @@ def cache_image(_id,
         
         url = image_cache_host + 'hh_' + size + '/' + _id[:3] + '/' + _id + '.jpg'
         
-        hsh = hashlib.md5(image_base64).hexdigest()
+        hsh = hashlib.md5(image_origin.encode('utf-8')).hexdigest()
```

Comment:
Yeah, I'd rather it we switched back to content-based hashing too :)

Also perhaps we shouldn't make add Client-specific features in this function? It'll be used across several parts in the Indexer pipeline, and not only being written to by the Client.